cmd/edit/edit.go

```diff
@@ -0,0 +1,45 @@
+package edit
+
+import (
+	"fmt"
+	"io/ioutil"
+	"encoding/json"
+
+	"../../utils"
```

Comment:
I think the program should run with absolute path if you:

`go get github.com/dnote-io/cli` and install all its dependencies by running `go get -d ./...` from the project root.

If the above works, we can remove the commented out relative paths.